integ-test/src/test/java/org/opensearch/sql/legacy/SQLIntegTestCase.java

```diff
@@ -404,7 +404,7 @@ protected static JSONObject getAllClusterSettings() throws IOException {
     return new JSONObject(executeRequest(request));
   }
 
-  protected static class ClusterSetting {
+  public static class ClusterSetting {
```

Comment:
ditto

Reply:
I use it in correctness test, which does not inherit this class.